conda/recipes/rmm/meta.yaml

```diff
@@ -14,13 +15,14 @@ source:
 
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: cuda_{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}_{{ cudaMallocAsync }}
   script_env:
     - RMM_BUILD_NO_GPU_TEST
     - VERSION_SUFFIX
     - CC
     - CXX
     - CUDAHOSTCXX
+    - BUILD_FLAGS="{{ build_flags }}"
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/rapidsai/rmm/pull/910/commits/52d3dc1fd73ed2ee39b76ce4c07f96497ba1eb78